src/main/webapp/xml/xml2json.xslt

```diff
@@ -50,14 +50,20 @@
          <xsl:text>"component" : "</xsl:text>
          <xsl:value-of select="component" />
          <xsl:text>",</xsl:text>
-         <xsl:text>"labels" : "</xsl:text>
+         <xsl:text>"labelsFlat" : "</xsl:text>
```

Comment:
Yes - the concatenated string for using the label filter - the array for display purposes.